kilkari/src/main/java/org/motechproject/nms/kilkari/domain/Subscription.java

```diff
@@ -80,6 +79,15 @@ public void setDeactivationReason(DeactivationReason deactivationReason) {
         this.deactivationReason = deactivationReason;
     }
 
+    /**
+     * Returns the weekId corresponding to today's date.
+     * @return
+     */
+    @Ignore int todaysWeekId() {
```

Comment:
Ok, makes sense. Any reason why we wouldn't have an access modifier on the field? (public/private)